tests/integration/client/feedback/dataset/local/test_dataset.py

```diff
@@ -72,15 +72,17 @@ def test_create_dataset_with_span_questions(argilla_user: "ServerUser") -> None:
 
     ds = FeedbackDataset(
         fields=[TextField(name="text")],
-        questions=[SpanQuestion(name="spans", field="text", labels=["label1", "label2"])],
+        questions=[SpanQuestion(name="spans", field="text", labels=["label1", "label2"], allow_overlapping=True)],
```

Comment:
- Should we separate these tests so that we have with/without overlap?
- Should we add records to both that have overlaps or not?
- Should we assert raises of the overlap error code?

Reply:
Thanks. I've created a new separate test checking possible values for the `allow_overlapping` flag. 

For related records questions, I will include some integration tests. Validations for responses and suggestions are not covered on this SDK, we will cover them in the new SDK